packages/shadcn/src/utils/transformers/transform-import.ts

```diff
@@ -35,8 +35,8 @@ function updateImportAliases(moduleSpecifier: string, config: Config) {
 
   // Not a registry import.
   if (!moduleSpecifier.startsWith("@/registry/")) {
-    // We fix the alias an return.
-    const alias = config.aliases.components.charAt(0)
+    // We fix the alias and return.
+    const alias = config.aliases.components.split("/")[0]
```

Comment:
This change has caused a regression where the import of `cn` is messed up in monorepos.

It results in
```ts
import { cn } from "@my-custom-repo/lib/utils"
```
instead of 
```ts
import { cn } from "@my-custom-repo/ui/lib/utils"
```
Although it works fine for the default initialized workspace - `@workspace/ui/lib/utils`

CC: @shadcn